inf-ruby.el

```diff
@@ -141,6 +141,7 @@ the same directory and used the same base name."
     #'identity
     '("\\(^%s> *\\)"                      ; Simple
       "\\(^(rdb:1) *\\)"                  ; Debugger
+      "\\(^(rdbg\\)"                      ; Ruby Debug Gem
```

Comment:
Why not `\\(^(rdbg)\\)`? Does the prompt ever look contain anything other than `(rdbg)`?

Reply:
Most of the times the prompt is simply `(rdbg) ` but when I run rails system tests with rspec - something about the spinning up of the webdriver process changes the prompt, in my case to `(rdbg@/Users/justintalbott/.asdf/installs/ruby/3.1.2/bin/rspec#40239) `, hence why I omitted the closing paren. The regex could look for a trailing `)` or `@`, but I didn't know if there were potentially other suffixes and I figured `(rdbg` was simpler. Let me know if you have a preference though.